src/content/en/updates/2018/12/get-installed-related-apps.md

```diff
@@ -184,13 +180,14 @@ your domain. Be sure to include the escaping characters.
 ### Test for the presence of your native app {: #test-native }
 
 Once you’ve updated your native app and added the appropriate fields to the
-web app manifest, you can add the code to check for the presence of your native
-app to you web app. Calling `navigator.getInstalledRelatedApps()` returns a
-`Promise` that resolves with an array of your apps that are installed on the
+web app manifest, add code to check for the presence of your native
+app to your web app. Calling `navigator.getInstalledRelatedApps()` returns a
+`promise` that resolves with an array of your apps that are installed on the
 user's device.
 
 ```js
-navigator.getInstalledRelatedApps().then((relatedApps) => {
+navigator.getInstalledRelatedApps()
+.then((relatedApps) => {
```

Comment:
I believe this is the more usual way of doing `.then()` these days. They will probably ask you to use `await`.